oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+product-attribute https://github.com/OCA/product-attribute 14.0
```

Comment:
FTR this file is useless for OCA builds since a while (unless you rely on it for your CI)